tests/playwright-tests/admin/azpage/AzureLoginPage.ts

```diff
@@ -0,0 +1,33 @@
+import { Locator, Page } from '@playwright/test';
+import environment from '../../utils/env';
+
+export default class AzureLoginPage {
+  readonly page: Page;
+  readonly emailAddress: Locator;
+  readonly nextButton: Locator;
+  readonly password: Locator;
+  readonly signInButton: Locator;
+  readonly noButton: Locator;
+
+  constructor(page) {
```

Comment:
Think to avoid any warnings from the compiler or IDE, it'd be good to have an explicit type here i.e.

```ts
constructor(page: Page)
```

Would suggest doing this for the other page classes as well